sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 3.0.0 (Unreleased)
+
+### Breaking Changes
```

Comment:
```suggestion
### Features Added

- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
  - added a type `MicrosoftTeamsAppKind`
  - added a method `isMicrosoftTeamsAppIdentifier` to check if the identifier is `MicrosoftTeamsAppIdentifier`.
  - added a field `teamsAppId` to `MicrosoftTeamsAppIdentifier`
```

I think everything should go under the "Features Added" and we should only mention public APIs